main.py

```diff
@@ -1,10 +1,44 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players_info = json.load(players_file)
+
+    for player, player_info in players_info.items():
+        guild = player_info["guild"] if player_info["guild"] else None
+        if guild:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild["name"],
+                description=guild["description"]
+            )
+
+        race = player_info["race"]
+        if race:
```

Comment:
You don't need this check; race always exists and even if it is not table in the db will not accept you create row without race or null in this column.